taskcluster/translations_taskgraph/transforms/dependency_dummies.py

```diff
@@ -1,8 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This transform sequence is used to workaround the fact that Taskcluster only
```

Comment:
Praise: This is the kind of thing that docs are really nice to have for. Thanks for taking the time for writing this and the others.